cpp/test/Ice/ami/AllTests.cpp

```diff
@@ -2482,6 +2485,28 @@ allTests(Test::TestHelper* helper, bool collocated)
 
     if(p->ice_getConnection())
     {
+        cout << "testing back pressure... " << flush;
+        {
+            // Keep the 3 server thread pool threads busy.
+            auto sleep1Future = p->sleepAsync(1000);
+            auto sleep2Future = p->sleepAsync(1000);
+            auto sleep3Future = p->sleepAsync(1000);
+
+            auto onewayProxy = Ice::uncheckedCast<Test::TestIntfPrx>(p->ice_oneway());
+
+            // Sending should be canceled because the TCP receive buffer size on the server is set to
```

Comment:
"canceled" doesn't sound appropriate here. I would use "block" or "not complete".